File: pdf_focus_net/PDF Focus .Net Trial (6.9.7.6)/Code samples/CSharp/PDF to HTML/Convert PDF to separate HTML pages/Sample.cs
using System;
using System.IO;

namespace Sample
{
    class Sample
    {
        static void Main(string[] args)
        {
            // Convert PDF to separate HTMLs.
            // Each PDF page will be converted to a single HTML document.

            // Path to PDF file.
            string pdfPath = Path.GetFullPath(@"..\..\..\..\..\simple text.pdf");

            // Directory to store HTML documents.
            string htmlDir = Path.GetDirectoryName(pdfPath);
            
            SautinSoft.PdfFocus f = new SautinSoft.PdfFocus();

            f.HtmlOptions.IncludeImageInHtml = false;
            // Path (must exist) to a directory to store images after converting. Notice also to the property "ImageSubFolder".
            f.HtmlOptions.ImageFolder = htmlDir;

            f.OpenPdf(pdfPath);

            // Convert each PDF page to separate HTML document.
            // simple text.html, simple text.html ... simple text.html.
            for (int page = 1; page <= f.PageCount; page++)
            {
                f.HtmlOptions.Title = String.Format("Simple Text - Page {0}", page);
                f.HtmlOptions.ImageSubFolder = String.Format("images-page{0}", page);
                string htmlString = f.ToHtml(page, page);

                // Save htmlString to file
                string htmlPath = Path.Combine(htmlDir, String.Format("Page{0}.html", page));
                File.WriteAllText(htmlPath, htmlString);

                // Let's show first and last HTML pages.
                if (page == 1 || page==f.PageCount)
                    System.Diagnostics.Process.Start(htmlPath);
            }

        }
    }
}

File: pdf_focus_net/PDF Focus .Net Trial (6.9.7.6)/Code samples/VB.Net/PDF to HTML/Convert PDF to separate HTML pages/Sample.vb
Imports System.IO
Imports System.Drawing.Imaging
Imports System.Collections.Generic
Imports SautinSoft

Module Sample

    Sub Main()
        ' Convert PDF to separate HTMLs.
        ' Each PDF page will be converted to a single HTML document.

        ' Path to PDF file.
        Dim pdfPath As String = Path.GetFullPath("..\..\..\..\simple text.pdf")

        ' Directory to store HTML documents.
        Dim htmlDir As String = Path.GetDirectoryName(pdfPath)

        Dim f As New SautinSoft.PdfFocus()

        f.HtmlOptions.IncludeImageInHtml = False
        ' Path (must exist) to a directory to store images after converting. Notice also to the property "ImageSubFolder".
        f.HtmlOptions.ImageFolder = htmlDir

        f.OpenPdf(pdfPath)

        ' Convert each PDF page to separate HTML document.
        ' simple text.html, simple text.html ... simple text.html.
        For page As Integer = 1 To f.PageCount
            f.HtmlOptions.Title = String.Format("Simple Text - Page {0}", page)
            f.HtmlOptions.ImageSubFolder = String.Format("images-page{0}", page)
            Dim htmlString As String = f.ToHtml(page, page)

            ' Save htmlString to file
            Dim htmlPath As String = Path.Combine(htmlDir, String.Format("Page{0}.html", page))
            File.WriteAllText(htmlPath, htmlString)

            ' Let's show first and last HTML pages.
            If page = 1 OrElse page = f.PageCount Then
                System.Diagnostics.Process.Start(htmlPath)
            End If
        Next page
    End Sub
End Module
